internal/stage_index_scan.go

```diff
@@ -31,8 +76,8 @@ func testIndexScan(stageHarness *test_case_harness.TestCaseHarness) error {
 
 	_ = os.Remove("./test.db")
 
-	if err := exec.Command("cp", path.Join(os.Getenv("TESTER_DIR"), "companies.db"), "./test.db").Run(); err != nil {
-		logger.Errorf("Failed to create test database, this is a CodeCrafters error.")
+	if err := os.Symlink(path.Join(os.Getenv("TESTER_DIR"), "companies.db"), "./test.db"); err != nil {
```

Comment:
I'm not sure if all user implementations follow symlinks by default, how about creating a hard link instead? (`os.Link`)